source/common/stats/BUILD

```diff
@@ -76,6 +78,17 @@ envoy_cc_library(
     ],
 )
 
+envoy_cc_library(
+    name = "scope_prefixer_lib",
+    srcs = ["scope_prefixer.cc"],
+    hdrs = ["scope_prefixer.h"],
+    deps = [
+        ":symbol_table_lib",
+        ":utility_lib",
+        "//include/envoy/stats:stats_interface",
+    ],
+)
```

Comment:
In the interest of making this PR smaller, this could be submitted separately (I think).

Reply:
It's already submitted I think.